src/cmd/troubleshoot/errors/aggregated.go

```diff
@@ -0,0 +1,31 @@
+package errors
+
+import (
+	"strings"
+)
+
+type AggregatedError struct {
+	Errs []error
+}
+
+func (errors *AggregatedError) Add(err error) {
+	errors.Errs = append(errors.Errs, err)
+}
+
+func NewAggregatedError() AggregatedError {
+	return AggregatedError{Errs: []error{}}
+}
+
+func (e AggregatedError) Error() string {
+	sb := strings.Builder{}
+	for _, err := range e.Errs {
+		sb.WriteString(err.Error() + "\n")
+	}
+	return sb.String()
```

Comment:
I'll allow `sb` as a name as long as there is some whitespace to make it more readable
```suggestion
	sb := strings.Builder{}
	
	for _, err := range e.Errs {
		sb.WriteString(err.Error() + "\n")
	}
	
	return sb.String()
```

Reply:
I have to say, I find the wording ("I will allow") here quite pretentious. And I don't think that the empty lines here add much for readability. They just bloat up the code and disconnect parts from each other that belong together.

Reply:
it was meant as a joke since I am usually rather strict concerning variable naming, I guess that doesn't translate well when written.

Reply:
I agree with Stefan here, this method does literally one thing and there is no need to separate its logic with empty lines.